src/Themes/HackathonTheme/Views/OrchardCore.Users/Account/Login.cshtml

```diff
@@ -22,73 +22,71 @@
+  @if (!disableLocalLogin)
+  {
+    <v-card-actions>
+      @await RenderSectionAsync("AfterLogin", required: false)
+      <v-btn type="submit" form="form" x-large block color="success">@T["Log in"]</v-btn>
+    </v-card-actions>
+  }
+  @if (loginProviders.Count == 0)
+  {
+    <div class="d-flex align-center">
+      <v-divider></v-divider><span class="px-2">@T["or"]</span><v-divider></v-divider>
+    </div>
+    <v-card-actions>
+      <v-row>
+        <v-col>
+          <form asp-controller="Account" asp-action="ExternalLogin" asp-route-returnurl="@ViewData["ReturnUrl"]" method="post">
+            @foreach (var provider in loginProviders)
+            {
+              <v-btn type="submit" x-large block name="provider" value="@provider.Name" title="Log in using your @provider.DisplayName account">@T["Log in using"] @provider.DisplayName</v-btn>
+            }
+          </form>
         </v-col>
-          @if (allowResetPassword)
-          {
-            <div class="mt-3">
-              <a asp-controller="ResetPassword" asp-action="ForgotPassword" class="label-link">@T["Forgot your password?"]</a>
-            </div>
-          }
-          @if (userCanRegister)
-          {
-            <div class="mt-3">
-              <a asp-controller="Registration" asp-action="Register" asp-route-returnurl="@ViewData["ReturnUrl"]">@T["Register as a new user"]</a>
-            </div>
-          }
       </v-row>
-    </form>
-    }
-    @if (loginProviders.Count > 0)
-    {
-      @* TODO *@
-        <div class="col-md-6 @(disableLocalLogin ? "offset-md-3" : String.Empty)">
-            <section>
-                <h4>@T["Use another service to log in."]</h4>
-                <hr />
-                <form asp-controller="Account" asp-action="ExternalLogin" asp-route-returnurl="@ViewData["ReturnUrl"]" method="post" class="form-horizontal">
-                    <div>
-                        <p>
-                            @foreach (var provider in loginProviders)
-                            {
-                                <button type="submit" class="btn btn-primary @(disableLocalLogin ? "w-100" : String.Empty)" name="provider" value="@provider.Name" title="Log in using your @provider.DisplayName account">@provider.DisplayName</button>
-                                @if (disableLocalLogin)
-                                {
-                                    <br />
-                                }
-                            }
-                        </p>
-                    </div>
-                </form>
-            </section>
-        </div>
-    }
-  </v-card-text>
+    </v-card-actions>
+  }
+  @if (userCanRegister)
+  {
+    <div class="d-flex align-center">
+      <v-divider></v-divider><span class="px-2">@T["or"]</span><v-divider></v-divider>
+    </div>
+    <v-card-actions>
+      <form id="RegistrationForm" asp-controller="Registration" asp-action="Register" asp-route-returnurl="@ViewData["ReturnUrl"]" method="post"></form>
```

Comment:
This should be a link.

Reply:
I think the ReturnlUrl functionality is needed to return to the login screen after registering. A link wouldn't give this without some extra param tacked on...

Reply:
I thought it was a link before ? `  <a asp-controller="Registration" asp-action="Register" asp-route-returnurl="@ViewData["ReturnUrl"]">@T["Register as a new user"]</a>`